### ISSUE TITLE
SNVPhyl no-longer works in Galaxy

### ISSUE THREAD

**Issue by apetkau** [phac-nml/irida#1018]
## Describe the bug
Installing the SNVPhyl pipeline tools and running the workflow (version `1.2.2`) no-longer works in Galaxy (tested on Galaxy `20.05` and `21.01`). The following exception is generated:

```
ca.corefacility.bioinformatics.irida.exceptions.WorkflowException: GalaxyResponseException{status=400, responseBody={"err_msg": {"sec_filter|select_genotype": "para
meter 'select_genotype': an invalid option ('__none__') was selected (valid options: het,^het,^miss,miss,hom,^hom) Using default: 'None'."}, "err_code": 0, "err_dat
a": {"9": {"sec_filter|select_genotype": "parameter 'select_genotype': an invalid option ('__none__') was selected (valid options: het,^het,^miss,miss,hom,^hom) Usi
ng default: 'None'."}}}, errorMessage=null, errorCode=null, traceback=null}
```

## Steps to reproduce the problem
What were you doing when you encountered the problem?
1. Install the SNVPhyl tool suite
2. Run the SNVPhyl pipeline.
3. The above exception occurs.

## Expected behaviour
I expected the pipeline to finish successfully.

## Additional context
None.

**Comment by apetkau** [comment]
It looks like the revision used by the workflow for the **bcftools_view** (`c4a9b38b435d`) is no longer available in Galaxy. The wrapper has been updated and one of the parameters for the workflow (`select_genotype`) is now invalid. This will likely need to be updated.

**Comment by tom114** [comment]
Fixed in #1035
